_overviews/cheatsheets/index.md

```diff
@@ -464,25 +460,26 @@ println(upper.getOrElse(""))</code></pre></td>
   case Some(x) =&gt; f(x)
   case None    =&gt; None
 }</code></pre></td>
-      <td>Same as map but function must return an optional value</td>
+      <td>Same as map but function must return an optional value.</td>
     </tr>
     <tr>
       <td><pre class="highlight"><code>optionOfOption.flatten</code></pre>
       <em><strong>same as</strong></em>
       <pre class="highlight"><code>optionOfOption match {
-  case Some(Some(x)) =&gt; Some(x)
-  case _             =&gt; None
+  case Some(x: Option[_]) =&gt; x
+  case None               =&gt; None
```

Comment:
This alternate pattern match for `flatten` is also true, but I predict the original was better at showing what `flatten` does.